romanesco/format/graph/networkx_to_clique_json.py

```diff
@@ -0,0 +1,42 @@
+import json
+import networkx as nx
+
+output = []
+
+# There is no clearly defined notion of undirected graphs in clique.json
+# It simply checks for edges going in either direction, so we ignore
+# whether it is an nx.DiGraph or just nx.Graph
+
+for (node, data) in input.nodes(data=True):
+    clique_node = {
+        '_id': {
+            '$oid': node
```

Comment:
Using ``node`` as the OID won't work in general, because Mongo expects a 24-character hex string for this OID value.  The simplest way to make this right is to hash ``node`` to generate an appropriate string and use that as the OID.  This makes the fixup to lines 29 and 32 below simpler (``u`` and ``v`` will just be replaced with the same hashing strategy used on ``node``).  Converting from Clique to NetworkX would simply copy over the OID to serve as the node label.

This means that the ``node`` label on the NetworkX graph will be ephemeral but this seems to be an acceptable tradeoff, at least for now.

(Dan and I just discussed this in person; leaving this note here for everyone else.)